packit/distgit.py

```diff
@@ -473,6 +473,7 @@ def create_bodhi_update(
             kerberos_realm=self.config.kerberos_realm,
```

Comment:
Here is done some setup when getting the Bodhi client. Is it enough in `packit/api.py` change to use the defaults?

Reply:
very good catch, push_update needed these to be properly authenticated